ctapipe/tools/dump_triggers.py

```diff
@@ -23,31 +22,28 @@ class DumpTriggersTool(Tool):
     # configuration parameters:
     # =============================================
 
-    infile = Path(exists=True, directory_ok=False, help="input simtelarray file").tag(
-        config=True
-    )
+    input_path = Path(
+        exists=True, directory_ok=False, help="input simtelarray file", allow_none=False
+    ).tag(config=True)
 
-    outfile = Path(
+    output_path = Path(
         default_value="triggers.fits",
         directory_ok=False,
         help="output filename (*.fits, *.h5)",
     ).tag(config=True)
 
-    overwrite = Bool(False, help="overwrite existing output file").tag(config=True)
-
     # =============================================
     # map low-level options to high-level command-line options
     # =============================================
 
     aliases = Dict(
-        {"infile": "DumpTriggersTool.infile", "outfile": "DumpTriggersTool.outfile"}
+        {
+            "input_path": "DumpTriggersTool.input_path",
```

Comment:
Please change to `--input` and `--output` to be consistent with the other tools.